### PR TITLE
add nuklear/4.01.5

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **nuklear/4.01.5**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

https://github.com/conan-io/conan-center-index/issues/621

### CODE REVIEW

File: recipes/nuklear/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import os
+
+from conans import ConanFile, tools
+
+class NuklearConan(ConanFile):
+    name = "nuklear"
+    description = "A single-header ANSI C immediate mode cross-platform GUI library."
+    license = "MIT"
```
Comment (uilianries) [inline recipes/nuklear/all/conanfile.py]:
```suggestion
    license = ["MIT", "Unlicense"]
```

Comment (uilianries) [reply recipes/nuklear/all/conanfile.py]: https://github.com/Immediate-Mode-UI/Nuklear#license

File: recipes/nuklear/all/test_package/test_package.c [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#include <nuklear.h>
+
+int main() {
+  return 0;
```
Comment (madebr) [inline recipes/nuklear/all/test_package/test_package.c]: A little test or sample usage?

File: recipes/nuklear/all/conandata.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+sources:
+  "4.01.5":
```
Comment (uilianries) [inline recipes/nuklear/all/conandata.yml]: https://github.com/Immediate-Mode-UI/Nuklear/blob/bb7c2519a3981de793617eeca975ba8edec29971/package.json#L3

Okay!

File: recipes/nuklear/all/test_package/test_package.c [hunk 4]
```diff
@@ -0,0 +1,5 @@
+#include <nuklear.h>
+
+int main() {
```
Comment (uilianries) [inline recipes/nuklear/all/test_package/test_package.c]: It would be nice if you call any method from Nuklear.

**Pull request merged**